src/platform/silabs/efr32/BLEManagerImpl.cpp

```diff
@@ -606,23 +616,29 @@ CHIP_ERROR BLEManagerImpl::StartAdvertising(void)
 CHIP_ERROR BLEManagerImpl::StopAdvertising(void)
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
-    sl_status_t ret;
 
     if (mFlags.Has(Flags::kAdvertising))
     {
+        sl_status_t ret = SL_STATUS_OK;
```

Comment:
i don't see `ret` used.

Reply:
Varialbe is used at line 627 which does show in the PR diff. Variable was moved from the top of the function to inside the if since it is only used there.